pkg/lint/runner.go

```diff
@@ -192,20 +192,34 @@ func (r Runner) Run(ctx context.Context, linters []*linter.Config, lintCtx *lint
 	sw := timeutils.NewStopwatch("linters", r.Log)
 	defer sw.Print()
 
-	var issues []result.Issue
+	var (
+		linterErr     error
+		failedLinters []string
+		issues        []result.Issue
+	)
+
 	for _, lc := range linters {
 		lc := lc
 		sw.TrackStage(lc.Name(), func() {
 			linterIssues, err := r.runLinterSafe(ctx, lintCtx, lc)
 			if err != nil {
+				failedLinters = append(failedLinters, lc.Linter.Name())
 				r.Log.Warnf("Can't run linter %s: %v", lc.Linter.Name(), err)
+
 				return
 			}
 			issues = append(issues, linterIssues...)
 		})
 	}
 
-	return r.processLintResults(issues), nil
+	if len(failedLinters) > 0 {
+		linterErr = fmt.Errorf(
+			"one or more linters failed to run: %s",
+			strings.Join(failedLinters, ", "),
```

Comment:
Only nit: Do we really need fmt (formatting string here)?

Reply:
I think it's more readable and pretty common with errors I think since 1.13 when `%w` became the de facto standard for wrapping errors. This isn't wrapping though so I guess string concatenation is the alternative you're looking for?

```go
linterErr = errors.New("one or more linters failed to run: " + strings.Join(failedLinters, ", ")
```

Reply:
Yeah, pretty much it is!

Reply:
I don't feel super strong about it so I'll push a change!